src/shop-client.js

```diff
@@ -142,7 +142,7 @@ const ShopClient = CoreObject.extend({
    * ```
    *
    * @method fetchQuery
-   * @public
+   * @private
```

Comment:
I'm not sure if it's cool to mark public methods as private for the purpose of hiding them in the docs... not sure what best practice is here. @m-ux might have an opinion? It makes it clearer in the API docs but makes it confusing in the source.